stories/ng2-component-lab/button.component.exp.ts

```diff
@@ -1,154 +1,43 @@
-      <sdc-button  [color]="'negative'" >Button</sdc-button>
-    `
-    }).case('Negative Button Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button  [color]="'negative'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Warning Button', {
-        showSource: true,
-        template: `
-      <sdc-button [color]="'warning'">Button</sdc-button>
-    `
-    }).case('Warning Button Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [color]="'warning'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Primary Outline', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'">Button</sdc-button>
-    `
-    }).case('Primary Outline Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Gray Outline', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'gray'">Button</sdc-button>
-    `
-    }).case('Gray Outline Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'gray'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Positive Outline', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'positive'">Button</sdc-button>
-    `
-    }).case('Positive Outline Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'positive'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Negative Outline', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'negative'">Button</sdc-button>
-    `
-    }).case('Negative Outline Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline'" [color]="'negative'" [disabled]="true">Button</sdc-button>
-    `
-    })
-    .case('Primary Outline Rounded', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline-rounded'">Button</sdc-button>
-    `
-    }).case('Primary Outline Rounded Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline-rounded'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Gray Outline Rounded', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline-rounded'" [color]="'grey'">Button</sdc-button>
-    `
-    }).case('Gray Outline Rounded Disabled', {
-        showSource: true,
-        template: `
-      <sdc-button [sdcButtonStyle]="'outline-rounded'" [color]="'grey'" [disabled]="true">Button</sdc-button>
-    `
-    }).case('Positive Outline Rounded', {
+    .case('Primary Buttons : Large, Medium, Small', {
         showSource: true,
+        context:{
+          alert:alert1
```

Comment:
This is not used in the template.
Add (click) on each button, showing the button props in alert window.